From 9a3c21c19941fee90172c040d4ba1cd7bcd0fd06 Mon Sep 17 00:00:00 2001
From: Corey Bryant <corey.bryant@canonical.com>
Date: Fri, 27 Jan 2023 17:04:46 -0500
Subject: [PATCH 23/86] Revert "Bug#34401798 close client connection when
 server connection is closed by server [postfix]"

This reverts commit ab62007d9cb59626eada463ad627a05a80dcea09.
---
 router/tests/integration/test_routing_direct.cc | 12 +++---------
 1 file changed, 3 insertions(+), 9 deletions(-)

diff --git a/router/tests/integration/test_routing_direct.cc b/router/tests/integration/test_routing_direct.cc
index 862683e9166..c53b3c8d74f 100644
--- a/router/tests/integration/test_routing_direct.cc
+++ b/router/tests/integration/test_routing_direct.cc
@@ -1317,15 +1317,9 @@ TEST_P(ConnectionTest, classic_protocol_wait_timeout) {
   {
     auto ping_res = cli.ping();
     ASSERT_ERROR(ping_res);
-    EXPECT_THAT(
-        ping_res.error().value(),
-        AnyOf(2013,  // Lost connection to MySQL server during query
-                     //
-              4031   // The client was disconnected by the server because of
-                     // inactivity. See wait_timeout and interactive_timeout
-                     // for configuring this behavior.
-              ))
-        << ping_res.error();
+    EXPECT_EQ(ping_res.error().value(), 4031) << ping_res.error();
+    // The client was disconnected by the server because of inactivity. See
+    // wait_timeout and interactive_timeout for configuring this behavior.
   }
 }
 
-- 
2.37.2

